.gitignore

```diff
@@ -0,0 +1,3 @@
+dist/
```

Comment:
Please remove this file.

Reply:
how do you prevent accidental commits of compiled files?